pay-api/tests/unit/api/test_eft_short_names.py

```diff
@@ -745,108 +812,3 @@ def test_search_eft_short_names(session, client, jwt, app):
                       data_dict['single-linked']['short_name'],
                       data_dict['single-linked']['accounts'][0],
                       data_dict['single-linked']['statement_summary'][0])
-
-
-@pytest.mark.skip(reason='This needs to be re-thought, the create cfs invoice job should be handling receipt creation'
-                         'and creating invoice references when payments are mapped, '
-                         'it should wait until 6 pm before marking invoices as PAID'
-                         'Otherwise calls to CFS could potentially fail and the two systems would go out of sync.')
-def test_apply_eft_short_name_credits(session, client, jwt, app):
```

Comment:
Not needed here. As stated in the comment a bunch of what this tests here no longer belongs here.
New tests in test_eft_payment_actions.py added.